src/test/tests.ts

```diff
@@ -809,15 +805,16 @@ describe('Server', function () {
 
     setTimeout(() => {
       expect(connectionCallbackSpy.calledOnce).to.be.true;
-      expect(connectionCallbackSpy.getCall(0).args[0]).to.equal('Error');
+      expect(connectionCallbackSpy.getCall(0).args[0].message).to.equal('Error');
```

Comment:
Can u explain your idea please @DxCx ? We have set a new error handling so now the error being returned is: `{ message: 'Error' }` and not `'Error'` anymore :(